### ISSUE TITLE
TypeError upon API response

### ISSUE THREAD

**Issue by GaborCC** [openai/openai-python#306]
### Describe the bug

When executing the API call specified in the code snippet the following error occurs: 
[error.txt](https://github.com/openai/openai-python/files/10995454/error.txt)

Based on printouts, in some cases the API return with an ill-formed Python dictionary as below.

API response with **null** part of the dictionary:
{"error":{"message":"The response was filtered due to the prompt triggering Azure OpenAI’s content management policy. Please modify your prompt and retry. To learn more about our content filtering policies please read our documentation: https://go.microsoft.com/fwlink/?linkid=2198766",
              "type":**null**,"param":"prompt","code":"content_filter","status":400}}

### To Reproduce

Run the code snippet attached on the input 'doc.txt'.

### Code snippets

```Python
def prompt_openai(doc, max_tokens):

    prompt = f"""
    Create a summary of the text below.

    '{doc}'
    """
    try:
        response = openai.Completion.create(
        engine = openai.deployment,
        prompt = prompt,
        temperature = 1,
        max_tokens = max_tokens,
        top_p = 0.95,
        frequency_penalty = 0.5,
        presence_penalty = 0.5,
        stop = None#"<|im_end|>"
        )

    return response['choices'][0]['text'].replace('\n', '').replace(' .', '.').strip()
```

### OS

Windows 10

### Python version

Python 3.10.9 

### Library version

openai 0.27.2

**Comment by GaborCC** [comment]
Workaround is to add the is to ..\Lib\site-packages\openai\api_requestor.py
at line 619
           rcontent = result.content.decode("utf-8")

            if ":null," in rcontent:
                rcontent = rcontent.replace(':null,', ':"None",')
                rcontent = json.loads(rcontent)

**Comment by hallacy** [comment]
Any chance you could repro this with a curl command and show what response you're getting? I can't see the doc.txt file here and am unable to repro this.

It's also possible that https://github.com/openai/openai-python/pull/356 might help with this issue

**Comment by GaborCC** [comment]
Yeah, #356 definitely looks like the root cause of the problem. Below you can find the minimum viable query which striggers the error. It'll not come with curl. Happy to test a new release with fix for #356.

"query": """Summarize the following text: 'of Grozny to obtain drinking water and food.", "11. On 5 March 2000, at around 10 a.m., 
the applicant was distributing drinking water among the residents of the Oktyabrskiy District when a group of federal servicemen in
 two UAZ vehicles arrived and enquired as to how they could get to a certain street. The applicant and some other residents explained to them how 
 to find that street, but the commander of the group asked the applicant to come with them and show them the way. The applicant agreed. The applicant s
 ubmitted eyewitness statements by two residents of the Oktyabrskiy District confirming the above-mentioned episode.", "12. When the servicemen arrived 
 at the street they were looking for, the applicant asked them to let him out. Instead, the military hit the applicant in the kidneys and put a bag over 
 his head. They ordered him to be silent and delivered him to the Temporary Office of the Interior of the Oktyabrskiy District of Grozny ([\\u201cthe](file://u201cthe/) 
 Oktyabrskiy VOVD\\u201d, [\\u0432\\u0440\\u0435\\u043c\\u0435\\u043d\\u043d\\u044b\\u0439](file://u0432/u0440/u0435/u043c/u0435/u043d/u043d/u044b/u0439) [\\u043e\\u0442\\u0434\\u0435\\u043b](file://u043e/u0442/u0434/u0435/u043b) [\\u0432\\u043d\\u0443\\](file://u0432/u043d/u0443/)
    u0442\\u0440\\u0435\\u043d\\u043d\\u0438\\u0445 [\\u0434\\u0435\\u043b](file://u0434/u0435/u043b) [\\u041e\\u043a\\u0442\\u044f\\u0431\\u0440\\u044c\\u0441\\u043a\\u043e\\u0433\\u043e](file://u041e/u043a/u0442/u044f/u0431/u0440/u044c/u0441/u043a/u043e/u0433/u043e) 
    [\\u0440\\u0430\\u0439\\u043e\\u043d\\u0430](file://u0440/u0430/u0439/u043e/u043d/u0430) [\\u0433](file://u0433/). [\\u0413\\u0440\\u043e\\u0437\\u043d\\u043e\\u0433\\u043e](file://u0413/u0440/u043e/u0437/u043d/u043e/u0433/u043e)). According to the applicant, the officers who 
    apprehended him could not know his identity because they did not check his identity papers.", "13. At the Oktyabrskiy VOVD the officers intimidated and ill-treated the applicant for several hours. In particular, they severely beat him, cut his hair and forced him to chew and swallow it, pressed a red-hot nail into his hands, forehead, nostrils and tongue and carved a derogatory word [\\u201cChichik\\u201d](file://u201cchichik/u201d) on his forehead with a nail or knife.", "14. The officers also questioned the applicant, but made no written record of the interrogation. They asked the applicant where he had fought as a rebel fighter and why there was a lis'
    """

**Comment by hallacy** [comment]
We released v0.27.3 today which has the fix for #356. Can you take another look?
